### ISSUE TITLE
Please update code examples

### ISSUE THREAD

**Issue by Justin-Koningsberger** [sebbo2002/ical-generator#534]
Please add "cal.method(ICalCalendarMethod.REQUEST);" somewhere in your examples, without it ics files sent in attachments to outlook users show the filename as "not supported calendar message.ics", and don't show accept/decline buttons.

**Comment by sebbo2002** [comment]
Where exactly would you expect to find this code example? Feel free to submit a pull request if you would like to customize the documentation or the examples.

**Comment by Justin-Koningsberger** [comment]
Sure, will do! It was a very annoying bug to chase down, that is why I wanted to update the docs for other devs. All search results I found on google were caused by something else (recurrence patterns for instance). Running the files through an ics validator showed no errors. Also sending the same invitation to a gmail account worked as it should, and forwarding that invitation to outlook account also worked properly. After finding the solution it was easier to find something relevant in the search results. I found this: https://stack.payne.run/blog/2020/05/13/not-supported-calendar-message, number 5 there seems to be the cause.
